app/repositories/FileNotificationRepository.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import config.AppConfig
+import org.mongodb.scala.model.Indexes.ascending
+import org.mongodb.scala.model.{IndexModel, IndexOptions}
+import models.SDESNotificationRecord
+
+import javax.inject.Inject
+import models.notification.SDESNotification
+import uk.gov.hmrc.mongo.play.json.PlayMongoRepository
+import uk.gov.hmrc.mongo.MongoComponent
+import utils.Logger.logger
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.{ExecutionContext, Future}
+
+class FileNotificationRepository @Inject()(mongoComponent: MongoComponent,
+                                           appConfig: AppConfig)(implicit ec: ExecutionContext)
+  extends PlayMongoRepository[SDESNotificationRecord] (
+  collectionName = "SDESNotification",
```

Comment:
Could this be renamed to something more repository/database specific. e.g. "file-notifications" or "sdes-file-notifications"